src/main/java/org/spongepowered/mod/mixin/core/block/data/MixinTileEntity.java

```diff
@@ -43,6 +46,9 @@
     @Shadow
     public abstract BlockPos getPos();
 
+    @Shadow
+    public abstract void markDirty();
```

Comment:
Why add the shadowed method here and not in `MixinTileEntityBanner` where it's called?